packages/gasket-plugin-nextjs/generator/app/app-router/app/layout.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function RootLayout({ children }) {
+  return (
+    <html lang='en'>
```

Comment:
We shouldn't assume `lang='en'`

Reply:
Eslint looks for a value here `<html> elements must have the lang prop  jsx-a11y/html-has-lang`. 

What would be a good default for this? 

Would `en` be safe for the boilerplate code?